pages/join.html

```diff
@@ -35,7 +35,8 @@ <h2 id="instructors">Instructors</h2>
 
 <h2 id="discussion">Conversations</h2>
 <p>
-  You can join the conversation by following <a href="{{site.baseurl}}/blog/">our blog</a>
+  You can join the conversation by following <a href="{{site.baseurl}}/blog/">our blog</a>,
+  <a href="https://swc-slack-invite.herokuapp.com/">on Slack</a>,
```

Comment:
Please define a variable in `_config.yml` with the URL for the Slack and reference it here so that if/when the URL changes, we don't wind up with dead links.